SpatialGDK/Source/SpatialGDK/Private/SpatialView/ViewDelta.cpp

```diff
@@ -1,174 +1,590 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 
 #include "SpatialView/ViewDelta.h"
+#include "SpatialView/EntityComponentTypes.h"
+
+#include "Algo/StableSort.h"
+#include <algorithm>
 
 namespace SpatialGDK
 {
 
-void ViewDelta::AddOpList(OpList Ops, TSet<EntityComponentId>& ComponentsPresent)
+void ViewDelta::SetFromOpList(TArray<OpList> OpLists, EntityView& View)
 {
-	for (uint32 i = 0; i < Ops.Count; ++i)
+	Clear();
+
+	for (OpList& Ops : OpLists)
 	{
-		ProcessOp(Ops.Ops[i], ComponentsPresent);
+		const uint32 Count = Ops.Count;
+		Worker_Op* OpData = Ops.Ops;
+		for (uint32 i = 0; i < Count; ++i)
+		{
+			ProcessOp(OpData[i]);
+		}
 	}
-	OpLists.Add(MoveTemp(Ops));
+	OpListStorage = MoveTemp(OpLists);
+
+	PopulateEntityDeltas(View);
+}
+
+void ViewDelta::Clear()
+{
+  EntityChanges.Empty();
```

Comment:
Spacing is off here

Reply:
A missed that. Will do another scan for stuff like this.

Reply:
Found a few more things like this where field names were in the wrong style too. I think I've caught them all now.